.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Configuration for known file extensions
```

Comment:
not sure this file should be in git

Reply:
It has been commited by OCA-git-bot ± a month ago : https://github.com/OCA/hr/commit/aa0f8bf37fa6941054cf10cf3c80cc467ed9f574